modules/items/item-sheet-ffg.js

```diff
@@ -222,6 +222,12 @@ export class ItemSheetFFG extends ItemSheet {
       default:
     }
 
+    // Render the description if not initialized
+    // May happen on item import
+    if (!data.data.renderedDesc){
+      data.data.renderedDesc = PopoutEditor.renderDiceImages(data.data.description, {});
```

Comment:
I'm confused... this already exists on line 226 of item-sheet-ffg.js

https://github.com/StarWarsFoundryVTT/StarWarsFFG/blob/main/modules/items/item-sheet-ffg.js#L226

Reply:
Really strange... is git playing tricks on me? Will dig into it...

Reply:
Yep, didn't fetch from upstream - possible that the bug reporter in Discord made the same mistake

Reply:
Double checked just to be sure, the release zip for v1.7 definitely includes this change. 👍